test/image/mocks/basic_error_bar.json

```diff
@@ -20,6 +20,11 @@
         ],
         "visible": true
       },
+      "error_x": {
+        "type": "data",
+        "symmetric": false,
+        "visible": true
+      },
```

Comment:
added x errorbars with no data so that the image wouldn't change, but it would have failed before this fix in `plot_api_test`

Reply:
thanks for the comment :books: